resources/source-types/kafka_otlp.yaml

```
@@ -331,6 +331,21 @@ spec:
           operator: equals
           value: true
 
+    - name: log_encoding
+      label: Log Encoding
+      description: The encoding of the log payload received from Kafka.
```

This is nit picky, but I think we should refer to kafka payloads as "Events". The receiver is a subscriber to the topic. 
```
To publish (write) and subscribe to (read) streams of events
```
https://kafka.apache.org/intro

```suggestion
      description: The encoding of the log event pulled from the Kafka topic.
```